docs/requirements.txt

```diff
@@ -0,0 +1,5 @@
+PySide2>=5.15
+Qt.py>=1.3
+pysvn>=0.1
+Sphinx
+sphinx_rtd_theme>=1.0.0
```

Comment:
Both of these should be pinned so that we have reproducible docs. Also why is PySide required?

Reply:
I assume PySide is required because sphinx has to be able to import the things that it documents, and rez-gui would not import correctly without at least one of the GUI frameworks being available.

Could probably be removed if we want to remove docs for rez-gui?

Reply:
My question kind of implied that it should be removed and we should tell sphinx to ignore rez-gui. But that wasn't  clear at all... Sorry about that.